<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
 "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
  <head>
    <title>Abhishek Sarkar</title>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600" rel="stylesheet" type="text/css">
    <link href="main.css" rel="stylesheet" type="text/css">
  </head>

  <body class="hyphenate">
    <h1>Abhishek Sarkar</h1>

    <ul id="menu">
      <li><a href="#publications">Publications</a></li>
      <li><a href="https://scholar.google.com/citations?user=f0dcwU8AAAAJ&hl=en">Scholar</a></li>
      <li><a href="https://github.com/aksarkar">Github</a></li>
      <li><a href="https://orcid.org/0000-0002-4636-9255">ORCID</a>
      <li><a href="https://www.linkedin.com/in/aksarkarmit/">LinkedIn</a></li>
      <li><a href="cv.pdf">CV</a></li>
    </ul>

    <div id="about">
      <p>I am a postdoctoral scholar at
        the <a href="https://www.uchicago.edu/">University of Chicago</a>
        working with <a href="http://stephenslab.uchicago.edu/">Matthew
        Stephens</a>.</p>

      <p>I completed my Ph.D. at <a href="http://mit.edu">Massachusetts
          Institute of Technology</a> in the <a href="http://compbio.mit.edu">
          Computational Biology group</a> at
          the <a href="http://www.csail.mit.edu/">Computer Science and
          Artificial Intelligence Lab</a>, advised
          by <a href="http://mit.edu/~manoli">Manolis Kellis</a>.</p>
    </div>

    <div id="research">
      <h2>Research</h2>
      <p>My primary research interests are to develop new statistical models,
        develop efficient algorithms and robust software to fit those models,
        and to apply those models to identify and characterize the biological
        processes driving human development and disease..</p>

      <p>Currently, I am working on methods to analyze single-cell genomic data
        and gain insight into the interplay between stochastic gene expression,
        cell type/state, and genetic variation at quantitative trait loci.</p>

      <p>Previously, I developed methods to query how genetic variation which
        does not alter protein-coding sequences contributes to human disease.
        We integrate epigenomic information from
        the <a href="http://genome.ucsc.edu/ENCODE/">ENCODE</a>
        and <a href="khttp://www.roadmapepigenomics.org/">Roadmap
        Epigenomics</a> consortia to identify cell-type&ndash;specific
        enhancers enriched for disease associated variants, and additional
        information about the transcriptional regulatory network to dissect
        their mechanistic role in psychiatric, metabolic, and autoimmune
        disorders. we then develop efficient Bayesian methods to understand the
        architecture of common diseases: how many causal genetic variants are
        there, where in the genome do they reside, and how should we design
        studies to more rapidly discover them in the population?</p>

      <div id="publications">
        <h2 class="unnumbered">Selected publications</h2>
        <ul>
          <li><strong>Sarkar, A. K.</strong> & Stephens,
            M. <a href="https://dx.doi.org/10.1101/2020.04.07.030007"><q>Separating
            measurement and expression models clarifies confusion in single
            cell RNA-seq analysis.</q></a> <em>bioRxiv</em>, 2020.</li>
          <li>Gao Wang, <strong>Abhishek Sarkar</strong>, Peter Carbonetto, and
            Matthew
            Stephens. <a href="https://dx.doi.org/10.1111/rssb.12388"><q>A
            simple new approach to variable selection in regression, with
            application to genetic fine-mapping.</q></a> <em>Journal of the
            Royal Statistical Society: Series B (Statistical Methodology)</em>,
            2020</li>
          <li><strong>Abhishek Sarkar</strong>*, Po-Yuan Tung*, et
            al. <a href="http://dx.doi.org/10.1371/journal.pgen.1008045"><q>Discovery
            and characterization of variance QTLs in human induced pluripotent
            stem cells.</q></a> <em>PLOS Genetics</em>, 2019</li> <em>*Equal
            contribution</em>
          <li>Yongjin Park*, <strong>Abhishek K. Sarkar</strong>*, et
            al. <a href="http://biorxiv.org/content/early/2017/02/10/107623"><q>Multi-tissue
            polygenic models for transcriptome-wide association
            studies.</q></a> <em>biorXiv</em>, 2017 (in revision) <em>*Equal
              contribution</em></li>
          <li>Roadmap Epigenomics Consortium, et
            al. <a href="https://dx.doi.org/10.1038/nature14248"><q>Integrative
            analysis of 111 reference human
            epigenomes.</q></a> <em>Nature</em>, 2015.</li>
        </ul>
      </div>
      <a class="totop" href="#">(top)</a>
  </body>
</html>
