<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
          "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
  <head>
    <title>Abhishek Sarkar</title>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600" rel="stylesheet" type="text/css">
    <link href="main.css" rel="stylesheet" type="text/css">
  </head>

  <body class="hyphenate">
    <h1>Abhishek Sarkar</h1>

    <ul id="menu">
      <li><a href="#publications">Publications</a></li>
      <li><a href="cv.pdf">CV</a></li>
    </ul>

    <div id="about">
      <p>I am a Ph.D. student at <a href="http://mit.edu">Massachusetts
          Institute of Technology</a>. I currently
        work in the <a href="http://compbio.mit.edu"> Computational Biology
          group</a> with <a href="http://mit.edu/~manoli">Manolis Kellis</a> at
        the <a href="http://www.csail.mit.edu/">Computer Science and
          Artificial Intelligence Lab</a>.</p>
    </div>

    <div id="research">
      <h2>Research</h2>
      <p>My main research interests are complex trait genomics, regulatory
        genomics, and epigenomics. I work on algorithms and statistical methods
        to elucidate the genetic basis of human disease.</p>

      <p>My current work develops efficient Bayesian methods to understand the
        architecture of common diseases: how many causal genetic variants are
        there, where in the genome do they reside, and how should we design
        studies to more rapidly discover them in the population?</p>

      <p>Previously, I developed methods to query how genetic variation which
        does not alter protein-coding sequences contributes to human disease.
        We integrate epigenomic information from
        the <a href="http://genome.ucsc.edu/ENCODE/">ENCODE</a>
        and <a href="khttp://www.roadmapepigenomics.org/">Roadmap
          Epigenomics</a> consortia to identify cell-type&ndash;specific
        enhancers enriched for disease associated variants, and additional
        information about the transcriptional regulatory network to dissect
        their mechanistic role in psychiatric, metabolic, and autoimmune
        disorders.</p>

      <div id="publications">
        <h2 id="pre-prints" class="unnumbered">Pre-prints</h2>
        <ul>
          <li>Yongjin Park*, <strong>Abhishek K. Sarkar</strong>*, et al. <a href="http://biorxiv.org/content/early/2017/02/10/107623">“Multi-tissue polygenic models for transcriptome-wide association studies.”</a> <em>BioRxiv</em>. (2017) <em>*Equal contribution</em></li>
          <li>Kunal Bhutani*, <strong>Abhishek Sarkar*</strong>, et al. <a href="http://biorxiv.org/content/early/2017/02/14/108316">“Modeling prediction error improves power of transcriptome-wide association studies.”</a> <em>BioRxiv</em>. (2017) <em>*Equal contribution</em></li>
          <li>Felix Day, …, <strong>Abhishek K. Sarkar</strong>, et al. <a href="http://biorxiv.org/content/early/2016/09/23/076794">“Genomic analyses for age at menarche identify 389 independent signals and indicate BMI-independent effects of puberty timing on cancer susceptibility.”</a> <em>BioRxiv</em>. (2016) <em>Author 9/215</em> (accepted)</li>
          <li><strong>Abhishek K. Sarkar</strong>, Lucas D. Ward, Manolis Kellis. <a href="http://biorxiv.org/content/early/2016/04/11/048066">“Functional enrichments of disease variants across thousands of independent loci in eight diseases.”</a> <em>BioRxiv</em>. (2016) (in revision)</li>
          <li>Yaping Liu, <strong>Abhishek Sarkar</strong>, Manolis Kellis.  <a href="http://biorxiv.org/content/early/2016/04/15/048827">“Evidence of a recombination rate valley in human regulatory domains.”</a> <em>BioRxiv</em>. (2016) (in revision)</li>
        </ul>

        <h2 id="peer-reviewed-publications" class="unnumbered">Peer-reviewed publications</h2>
        <ul>
          <li>Roadmap Epigenomics Consortium et al. <a href="http://www.nature.com/nature/journal/v518/n7539/full/nature14248.html">“Integrative analysis of 111 reference human epigenomes.”</a> <em>Nature</em>, 518(7539), 317–330 (2015). <em>Integrative analysis lead (equal contributor)</em>.</li>
        </ul>

        <h2>Presentations (selected)</h2>
        <ul>
          <li><strong>Abhishek Sarkar</strong>, Yongjin Park, Manolis Kellis. “Dissecting the non-infinitesimal architecture of complex traits using group spike-and-slab priors” (contributed talk). Workshop on Machine Learning in Computational Biology, Thirtieth Annual Conference on Neural Information Processing Systems, Barcelona, Spain. 2016.</li>
          <li><strong>Abhishek Sarkar</strong>, Yongjin Park, Manolis Kellis. “Dissecting the non-infinitesimal architecture of complex traits” (poster). 68<sup>th</sup> meeting of the American Society of Human Genetics, Vancouver, Canada. 2016.</li>
          <li><strong>Abhishek Sarkar</strong>, Luke Ward, Manolis Kellis. “Functional enrichments of disease variants across thousands of independent loci in eight diseases.” (talk). Leena Peltonen School of Human Genomics, Wellcome Trust Sanger Institute, Hinxton, Cambridge, UK. 2016.</li>
        </ul>
      </div>

      <a class="totop" href="#">(top)</a>
  </body>
</html>
